.github/workflows/e2e-tests-pr.yml

```diff
@@ -6,6 +6,16 @@ on:
     branches:
       - 'master'
       - 'release/*'
+    paths-ignore:
+      - '.devcontainer/**'
+      - '.github/*'
+      - '.github/scripts/*'
+      - '.github/workflows/benchmark-*'
+      - '.github/workflows/check-*'
```

Comment:
Thought about `.github/**`, but thought maybe we want to still run the workflow when we are making changes to this specific workflow. Therefore ignored more specifically